scripts/functions.sh

```diff
@@ -554,11 +555,12 @@ function entry_point_script {
 
 function setup_monitoring() {
 
-if [[ -f "${EXTRA_CONFIG_DIR}"/monitor.properties ]]; then
-      envsubst < "${EXTRA_CONFIG_DIR}"/monitor.properties > "${GEOSERVER_DATA_DIR}"/monitoring/monitor.properties
-else
+  if [[ -f "${EXTRA_CONFIG_DIR}"/monitor.properties ]]; then
+        envsubst < "${EXTRA_CONFIG_DIR}"/monitor.properties > "${GEOSERVER_DATA_DIR}"/monitoring/monitor.properties
+  else
 
-cat > "${GEOSERVER_DATA_DIR}"/monitoring/monitor.properties <<EOF
+  if [[ ! -f "${GEOSERVER_DATA_DIR}"/monitoring/monitor.properties ]]; then
```

Comment:
I see, we have different use models, but I understand your point. I have reverted my changes that prevented overwriting the monitor.properties and logging.xml files. 

I have renamed some environment variables to better align with Geoserver documentation and defaults, and exposed them to the user by adding them to the .env file.

GEOSERVER_LOG_PROFILE is more accurate than GEOSERVER_LOG_LEVEL in this case.

Exposing GEOSERVER_LOG_DIR allows the user to customize log location. For example, we use an EFS share for GEOSERVER_DATA_DIR but local EBS volume for log files. Previously I have had to override the log location with an ADDITIONAL_JAVA_STARTUP_OPTION to accomplish this.

How do you feel about these changes? I suppose one concern would be backwards compatibility, in case a user had been utilizing the undocumented GEOSERVER_LOG_LEVEL. If you agree with the changes we could add a check for existence of that environment variable and use it.